test/known_problems_should_fail_SUITE.erl

```diff
@@ -0,0 +1,63 @@
+-module(known_problems_should_fail_SUITE).
```

Comment:
Did you duplicate the eunit test suites as commontest suite?

I don't want duplicated logic. Delete the eunit test suites that have been ported to commontest.

Reply:
I did, yes. If we're happy with moving completely to CT, I'll delete the EUnit tests.